USAGE.md

```diff
@@ -53,28 +53,22 @@ The `make-maps` command can be used to generate a small set of standardized, rep
 
 This command can be run according to the following usage:
 ```commandline
-Usage: make-maps [OPTIONS]
+Usage: reView-tools make-maps [OPTIONS]
 
-  Generates standardized, presentation-quality maps for the input supply
-  curve, including maps for each of the following attributes: Capacity
-  (capacity), All-in LCOE (total_lcoe), Project LCOE (mean_lcoe), LCOT (lcot),
-  Capacity Density (derived column) [wind only]
+  Generates standardized, presentation-quality maps for the input supply curve, including maps for each of the following attributes: Capacity (capacity), All-in LCOE (total_lcoe), Project LCOE (mean_lcoe), LCOT (lcot), Capacity Density (derived column) [wind only]
 
 Options:
-  -i, --supply_curve_csv FILE  Path to supply curve CSV file.  [required]
-  -t, --tech [wind|solar]      Technology choice for ordinances to export.
-                               Valid options are: ['wind', 'solar'].
-                               [required]
-  -o, --out_folder DIRECTORY   Path to output folder for maps.  [required]
-  -b, --boundaries FILE        Path to vector dataset with the boundaries to
-                               map. Default is to use state boundaries for
-                               CONUS from Natural Earth (1:50m scale), which
-                               is suitable for CONUS supply curves. For other
-                               region, it is recommended to provide a more
-                               appropriate boundaries dataset.
-  -d, --dpi INTEGER RANGE      Dots-per-inch (DPI) for output images. Default
-                               is 600.  [x>=0]
-  --help                       Show this message and exit.
+  -i, --supply_curve_csv FILE     Path to supply curve CSV file.  [required]
+  -S, --breaks-scheme TEXT        Scheme to use for deriving the breaks for classifying the supply curve variables. Valid options are: ['wind', 'solar'] or the name of any scheme from the mapclassify package(see https://pysal.org/mapclassify/api.html#classifiers) followed by optional keyword arguments (e.g., 'equalinterval:{"k": 10}'). If 'solar' or 'wind' are specified, hard-coded breaks will be used for each mapped variable based on the typical range of values for that variable and technology. Alternatively, if a mapclassify scheme is specified, that scheme will be used to dynamically set the breaks for all mapped variables. Must be specified unless the legacy --tech option is used instead.
```

Comment:
Did you intentionally move away from newlines here (i.e. put the entire help text into a single line)? If it's intentional, that's fine, but it makes the render a little more difficult to read since the users have to scroll back and forth horizontally